examples/automm/example_tabular.py

```diff
@@ -0,0 +1,36 @@
+from autogluon.text.automm import AutoMMPredictor
+import argparse
+from utils import get_tabular_data 
+
+def main(args):
+
+    train_df, val_df, test_df, column_types, problem_type, metric = get_tabular_data(
+        base_path=args.dataset_dir,
+        dataset_name=args.dataset_name,
+    )
+
+    predictor = AutoMMPredictor(
+        label='0', 
+        problem_type=problem_type,
+        eval_metric=metric,
+        path=args.exp_dir,
+        verbosity=4,
+    )
+
+    predictor.fit(
+        train_data=train_df,
+        tuning_data=test_df,
+        column_types=column_types,
```

Comment:
why are we passing column_types? Isn't the dataframe dtypes enough?

Reply:
Although `column_types` is optional, we found that very occasionally the column_types may be inferred errorly (for example some NUMERICAL columns are identified as CATEGORY). 
In our currently implementation, the model configurations for [NUMERICAL] and [CATEGORY,NUMERICAL] are different:
- [NUMERICAL]: model.numerical_transformer.num_trans_blocks = 3;
- [CATEGORY,NUMERICAL]: model.numerical_transformer.num_trans_blocks = 0, model.categorical_transformer.num_trans_blocks = 0, and model.fusion_transformer.num_trans_blocks = 3.
Hence, passing column_types is necessary.

We'll update `create_model` to make sure that fusion_transformer is applied to [CATEGORY,NUMERICAL], [NUMERICAL] and [CATEGORICAL]. Afterwards, passing `column_types` will not be necessary.